pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1102,6 +1102,10 @@ type GatewaySpec struct {
 	// +nullable
 	// +optional
 	ExternalRgwEndpoints []v1.EndpointAddress `json:"externalRgwEndpoints,omitempty"`
+
+	// KMSTokenSecretName is the kubernetes secret containing the KMS token
```

Comment:
Kubernetes is capitalized. Kubernetes docs capitalize "Secret". It is most helpful to not use acronyms in user-facing docs.

```suggestion
	// KMSTokenSecretName is the Kubernetes Secret containing the Key Management Service token.
```